src/ServicePulse.Host/vue/src/components/configuration/HealthCheckNotifications_ConfigureEmail.vue

```diff
@@ -32,12 +32,20 @@ const fromIsValid = computed(() => {
   return from.value && emailRe.test(from.value) ? true : false;
 });
 const toIsValid = computed(() => {
-  return to.value && emailRe.test(to.value) ? true : false;
+    return to.value && validateMultipleEmailsSemicolonSeparated(to.value) ? true : false;
 });
 const formIsValid = computed(() => {
   return smtpServerIsValid.value && smtpPortIsValid.value && fromIsValid.value && toIsValid.value;
 });
 
+function validateMultipleEmailsSemicolonSeparated(value) {
+    var result = value.split(";");
+    for (var i = 0; i < result.length; i++)
+        if (!emailRe.test(result[i]))
+            return false;
+    return true;
+}
+
```

Comment:
```suggestion
function validateMultipleEmailsSemicolonSeparated(value) {
  var result = value.split(",");
  for (var i = 0; i < result.length; i++)
    if (!emailRe.test(result[i]))
      return false;
  return true;
}

```

It's a [comma](https://learn.microsoft.com/en-us/dotnet/api/system.net.mail.mailmessage.-ctor?view=net-7.0#system-net-mail-mailmessage-ctor(system-string-system-string)) not a semicolon.